lib/ansible/modules/user.py

```diff
@@ -3114,6 +3120,18 @@ class BusyBox(User):
         - remove_user()
         - modify_user()
     """
+    def _build_password_string(self, current_password=None):
+        lock = '!' if self.password_lock else ''
```

Comment:
The `'!'` literal is used 3 times in this function. Perhaps, it's worth putting it into a named var?

Reply:
Oh, it's also in use outside, so maybe even a module-level constant.